api/src/main/java/org/openmrs/module/ugandaemrreports/library/DataFactory.java

```diff
@@ -968,6 +968,13 @@ public CohortDefinition getPatientsWhoseMostRecentObsDateIsBetweenValuesByEndDat
         return convert(cd, params);
     }
 
+    public CohortDefinition getLostClients() {
```

Comment:
@solemabrothers Lost is from 30 days to 89 days - so I think we need to set a maxiumum too as from 90 its Lost to Followup

Reply:
Ok